site/src/pages/docs/advanced-scala.mdx

```diff
@@ -2,6 +2,33 @@ import versions from '../../../gen-src/versions.json';
 
 # Scala integration
 
+As a matter of fact, there are three distinct ways to integrate Armeria and Scala:
+
+* by using the `armeria-scala` library;
+* by using the [`http4s-armeria`](https://github.com/http4s/http4s-armeria) library;
```

Comment:
In fact, we also have Tapir Armeria backend. https://tapir.softwaremill.com/en/latest/server/armeria.html
As you write ways to Armeria with Scala, it would be worth mentioning it. 😆

If it is, you think, out of the scope of this PR, I will add the section later.

Reply:
Uh-oh, sorry, I wasn't aware of it :( It'd great if you could expand this section!